public/js/main.js

```diff
@@ -101,3 +85,42 @@ function renderToolbox() {
     document.querySelector("#mount")
   );
 }
+
+if(process.env.NODE_ENV !== "DEVTOOLS_PANEL") {
```

Comment:
Thanks for the write up. You're rationale makes sense. 

Lets discuss this after your work lands.